pkg/encryption/master_key.go

```diff
@@ -0,0 +1,118 @@
+// Copyright 2020 TiKV Project Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package encryption
+
+import (
+	"encoding/hex"
+	"io/ioutil"
+	"strings"
+
+	"github.com/pingcap/errors"
+	"github.com/pingcap/kvproto/pkg/encryptionpb"
+	"github.com/tikv/pd/pkg/errs"
+)
+
+const (
+	// Master key is of fixed 256 bits (32 bytes).
+	masterKeyLength = 32 // in bytes
+)
+
+// MasterKey is used to encrypt and decrypt encryption metadata (i.e. data encryption keys).
+type MasterKey struct {
+	// Master key config. Used to compare if two master key is the same.
+	Config *encryptionpb.MasterKey
```

Comment:
Do we need to rename it to `MasterKeyConfig` in `kvproto`?

Reply:
I think both are okay?